go.mod

```diff
@@ -17,10 +17,13 @@ require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.7
 	github.com/digitalocean/go-qemu v0.0.0-20221209210016-f035778c97f7
 	github.com/diskfs/go-diskfs v1.7.0 // gomodjail:unconfined
+	github.com/docker/docker v28.3.3+incompatible
```

Comment:
Will it help with respect to code maintenance if we can create a go mod to handle all these.

I did create trackport to handle multiple various implementation of port identification 

https://github.com/balajiv113/trackport/blob/main/pkg/trackapi/api.go

https://github.com/balajiv113/trackport/tree/main/pkg/internal